### ISSUE TITLE
VTOL: standard VTOL climbs a lot during forward transition

### ISSUE THREAD

**Issue by AndreasAntener** [PX4/Firmware#3843]
Effect gets bigger in strong head wind.
Could potentially be improved by tighter MC altitude tuning. Might need an extra mode in the position or vtol controller to react faster to sudden wing lift.

**Comment by AndreasAntener** [comment]
More logs from Sander showing the in #3926

**Comment by AndreasAntener** [comment]
@sanderux @tumbili 
My guess is that the pusher together with the quad holding attitude create more lift than the mc pos controller compensates, or we don't control Z velocity anymore during the transition.
Later could have something todo with the following control flags set to false:
```
control_mode.flag_control_position_enabled
control_mode.flag_control_velocity_enabled
```

**Comment by sanderux** [comment]
![image](https://cloud.githubusercontent.com/assets/14801663/13549762/378fa8f4-e30d-11e5-8aa0-5e9c1f4a2474.png)

I think the pitch is being pushed up by the pusher and the quad compensation is too late

**Comment by sanderux** [comment]
![image](https://cloud.githubusercontent.com/assets/14801663/13549873/157f6c4c-e310-11e5-9900-84f1af074668.png)

on the other hand you might be right, the quad doesnt seem to throttle down much when altitude increases

**Comment by AndreasAntener** [comment]
yea I'm not yet sure. I got suspicious when I looked at the local position velocity setpoints during the transition. They seem to be reset. Didn't spot something obvious in the controller though.

**Comment by AndreasAntener** [comment]
Fixed on master in a62046625b234dcc71279d036b6a70530c67c030
